restapi/stories.go

```diff
@@ -77,8 +77,61 @@ func (api api) AddAuthorToStories(stories []models.Story) ([]models.Story, error
 
 func (api api) ReturnAllStories(w http.ResponseWriter, r *http.Request) render.Renderer {
 	var stories []models.Story
+	var storiesByTags []models.Tag
 
-	err := api.database.Where("is_visible = true").Find(&stories).Error
+	getVisibility := r.URL.Query()["visibility"]
```

Comment:
why not do r.FormValue["visibility"]. i think if it wont exist, it would default to ""

Reply:
These are query params, not form values. The reason I used from values for the ones below is so I could store the results inside arrays (which is not needed for this)